core/trino-spi/src/main/java/io/trino/spi/predicate/Range.java

```diff
@@ -381,14 +381,14 @@ public String toString(ConnectorSession session)
     {
         Object lowObject = lowValue
                 .map(value -> type.getObjectValue(session, nativeValueToBlock(type, value), 0))
-                .orElse("<min>");
```

Comment:
brackets were intentional, since varchar `'min'` is a possible valid min value.
Of course `'<min>'` is a potental value too, but i do not yet see how the change is an improvement.

Reply:
reverted this now, `<min>` is consistent with `PlanPrinter#formatDomain`